extensions/adobe/experience/adcloud/dsp/account.example.1.json

```diff
@@ -0,0 +1,10 @@
+{
+  "xdm:id": "12",
+  "xdm:accountKey": "dsdfweW12",
+  "xdm:accountName": "Dailymotion",
+  "xdm:accountType": "Brand Direct No Agency",
```

Comment:
The enum list value for accountType doesn't have spaces.
I'm not sure if there is any automated validation of these example files so this may not matter.

Reply:
Nevermind...my comment above...I didn't notice the meta:enum section.